jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ServiceHandlerTest.java

```diff
@@ -62,16 +58,16 @@ public void getServicesTest() {
         services = serviceHandler.getServices(serviceConfigs);
 
         //Assertion
-        assertNotNull(services);
-        assertEquals("TCP", services.get(0).getSpec().getPorts().get(0).getProtocol());
-        assertEquals("port-test", services.get(0).getSpec().getPorts().get(0).getName());
-        assertEquals(50, services.get(0).getSpec().getPorts().get(0).getNodePort().intValue());
-        assertEquals(80, services.get(0).getSpec().getPorts().get(0).getTargetPort().getIntVal().intValue());
-        assertEquals(8080, services.get(0).getSpec().getPorts().get(0).getPort().intValue());
-        assertEquals("NodePort", services.get(0).getSpec().getType());
-
-        assertTrue(services.get(0).getMetadata().getLabels().get("expose").equalsIgnoreCase("true"));
-        assertTrue(services.get(0).getMetadata().getName().equalsIgnoreCase("service-test"));
+        assertThat(services).isNotNull();
+        assertThat(services.get(0).getSpec().getPorts().get(0).getProtocol()).isEqualTo("TCP");
+        assertThat(services.get(0).getSpec().getPorts().get(0).getName()).isEqualTo("port-test");
+        assertThat(services.get(0).getSpec().getPorts().get(0).getNodePort().intValue()).isEqualTo(50);
+        assertThat(services.get(0).getSpec().getPorts().get(0).getTargetPort().getIntVal().intValue()).isEqualTo(80);
+        assertThat(services.get(0).getSpec().getPorts().get(0).getPort().intValue()).isEqualTo(8080);
+        assertThat(services.get(0).getSpec().getType()).isEqualTo("NodePort");
+
+        assertThat(services.get(0).getMetadata().getLabels().get("expose")).isEqualToIgnoringCase("true");
+        assertThat(services.get(0).getMetadata().getName()).isEqualToIgnoringCase("service-test");
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.